webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -472,6 +476,42 @@ func (v validator) validateStorageClass(ctx *pkgctx.WebhookRequestContext, vm *v
 	return append(allErrs, field.Invalid(scPath, scName, fmt.Sprintf(storageClassNotAssignedFmt, vm.Namespace)))
 }
 
+func (v validator) validateCrypto(
+	ctx *pkgctx.WebhookRequestContext,
+	vm *vmopv1.VirtualMachine) field.ErrorList {
+
+	var (
+		allErrs      field.ErrorList
+		path         = field.NewPath("spec", "crypto", "className")
```

Comment:
Should it be `encryptionClassName` at the end? This is not captured in test since you hard-coded the field path with this incorrect name.